src/main/java/dev/jbang/cli/BaseBuildCommand.java

```diff
@@ -472,9 +472,7 @@ protected static void searchForMain(ScriptSource src, RunContext ctx, File tmpJa
 																			STRINGTYPE) != null)
 															.findFirst();
 
-					if (agentmain.isPresent()) {
```

Comment:
I'm okay with the other commits in this PR, but personally I find this harder to read than the original code.

Reply:
Dropped the commit